projects/github-actions/required-review/src/requirement.js

```diff
@@ -126,7 +124,35 @@ function buildReviewerFilter( config, teamConfig, indent ) {
 		};
 	}
 
-	// WTF?
+	if ( op === 'is-author-or-reviewer' ) {
+		return async function ( reviewers ) {
+			core.info( `${ indent }Author or reviewers are union of these:` );
+			const authorOrReviewers = [ ...reviewers, github.context.payload.pull_request.user.login ];
+			const reviewersAny = await Promise.all(
+				arg.map( f => f( authorOrReviewers, `${ indent }  ` ) )
+			);
+			const requirementsMet = [];
+			const neededTeams = [];
+			for ( const requirementResult of reviewersAny ) {
+				if ( requirementResult.teamReviewers.length !== 0 ) {
+					requirementsMet.push( requirementResult.teamReviewers );
+				}
+				if ( requirementResult.neededTeams.length !== 0 ) {
+					neededTeams.push( requirementResult.neededTeams );
+				}
+			}
+			if ( requirementsMet.length > 0 ) {
+				// If there are requirements met, zero out the needed teams
+				neededTeams.length = 0;
+			}
+			return printSet(
+				`${ indent }=>`,
+				[ ...new Set( requirementsMet.flat( 1 ) ) ],
+				[ ...new Set( neededTeams.flat( 1 ) ) ]
+			);
+		};
+	}
+
 	throw new RequirementError( `Unrecognized operation "${ op }"`, {
```

Comment:
Why don't we return early like we did before with the `default`?

Reply:
No real reason. I couldn't cover this line before because of the early return, and I wound up deciding that there was no harm in leaving the throw until the end here instead.